test/e2e/usersignup_test.go

```diff
@@ -281,6 +281,47 @@ func (s *userSignupIntegrationTest) TestProvisionToOtherClusterWhenOneIsFull() {
 	})
 }
 
+func (s *userSignupIntegrationTest) TestFillingUpClusterCapacityFlipsSPCsToNotReady() {
+	t := s.T()
+	hostAwait := s.Host()
+	memberAwait1 := s.Member1()
+	memberAwait2 := s.Member2()
+
+	spaceprovisionerconfig.UpdateForCluster(t, hostAwait.Awaitility, memberAwait1.ClusterName, testSpc.MaxNumberOfSpaces(1))
+	spaceprovisionerconfig.UpdateForCluster(t, hostAwait.Awaitility, memberAwait2.ClusterName, testSpc.Enabled(false))
+	hostAwait.UpdateToolchainConfig(t, testconfig.AutomaticApproval().Enabled(true))
+
+	t.Run("SPC with free capacity is ready", func(t *testing.T) {
+		// then
+		_, err := wait.For(t, hostAwait.Awaitility, &toolchainv1alpha1.SpaceProvisionerConfig{}).FirstThat(
+			assertions.Has(spaceprovisionerconfig.ReferenceToToolchainCluster(memberAwait1.ClusterName)),
+			assertions.Is(testSpc.Ready()))
+		require.NoError(t, err)
+
+		_, err = wait.For(t, hostAwait.Awaitility, &toolchainv1alpha1.SpaceProvisionerConfig{}).FirstThat(
+			assertions.Has(spaceprovisionerconfig.ReferenceToToolchainCluster(memberAwait2.ClusterName)),
+			assertions.Is(testSpc.NotReady()))
+		require.NoError(t, err)
+	})
+
+	t.Run("deploying a space fills up the member, flipping its SPC to not ready", func(t *testing.T) {
+		// when
+		user1 := NewSignupRequest(s.Awaitilities).
+			Username("fill-up-user-1").
+			Email("fillup1@redhat.com").
+			EnsureMUR().
+			RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedAutomatically())...).
+			Execute(s.T())
+
+			// then
+		VerifyResourcesProvisionedForSignup(t, s.Awaitilities, user1.UserSignup)
```

Comment:
it's not really obvious, but calling the `EnsureMUR` triggers verification of the provisioned resources for the signup, so no need to do it for the second time.
apart from that, verifying resources is not important in this case so you can just wait until the MUR is present and slightly speed the test up:
```suggestion
			WaitForMUR().
			RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedAutomatically())...).
			Execute(s.T())
			
		// then
```